docs/Usage/Modules/BackToStart.md

```diff
@@ -1,4 +1,5 @@
 # BackToStart
+A module for generating an object's return trajectory.
 
 ## About the module
-This module is currently in development and documentation for this module will be released later.
\ No newline at end of file
+This module provides a service to reverse the test object's trajectory and generate a Williamson turn at the start and end to let the object return to its starting point. Execute the new trajectories similar to a normal test with `Arm` and `Start`. It is up to the operator to determine if the new trajectories are possible to execute or not. The service is only allowed in `Disarmed` state and calling `Reset test` in the GUI does not result in a state change. 
```

Comment:
"The service is only allowed in `Disarmed` state and calling `Reset test` in the GUI does not result in a state change."
 Sorry i don't understand this. A step by step guide on how the function is used after a test would be awesome!

Reply:
I included a step by step guide as well as two images showing what happens when pressing `Reset Test`, hopefully it's more clear now.